### PR TITLE
Updated GTA.Versions to 1.0.1868.0.

### PR DESCRIPTION
**Pull request by Tanjitsu**

### CODE REVIEW

File: source/scripting_v2/GTA/GameVersion.cs [hunk 1]
```diff
@@ -69,5 +69,7 @@ public enum GameVersion
 		VER_1_0_1737_0_NOSTEAM,
 		VER_1_0_1737_6_STEAM,
 		VER_1_0_1737_6_NOSTEAM,
+        VER_1_0_1868_0_STEAM,
```
Comment (crosire) [inline source/scripting_v2/GTA/GameVersion.cs]: Tabs please =)

Comment (justalemon) [reply source/scripting_v2/GTA/GameVersion.cs]: `.editorconfig` would be useful to prevent this in the future.

Comment (crosire) [reply source/scripting_v2/GTA/GameVersion.cs]: There is one already, so I don't know how this happened: https://github.com/crosire/scripthookvdotnet/blob/master/.editorconfig

Comment (justalemon) [reply source/scripting_v2/GTA/GameVersion.cs]: Didn't saw it. Maybe he used a Code Editor/IDE that does not supports `.editorconfig`

Comment (Tanjitsu) [reply source/scripting_v2/GTA/GameVersion.cs]: What is this sorcery of which you speak !

Comment (crosire) [reply source/scripting_v2/GTA/GameVersion.cs]: Fixed that in https://github.com/crosire/scripthookvdotnet/pull/901/commits/540251ec0a61278d9ce987e841c07ed467153d0b. Make sure you don't have the "Hide whitespace changes" option selected in GitHub. But I can also see it in my local Git client.

**Pull request merged**